executor/write.go

```diff
@@ -1919,7 +1926,7 @@ func (e *UpdateExec) exec(ctx context.Context, schema *expression.Schema) (types
 				continue
 			}
 			// Update row
-			changed, _, _, err1 := updateRecord(e.ctx, handle, oldData, newTableData, flags, tbl, false)
+			changed, _, _, _, err1 := updateRecord(e.ctx, handle, oldData, newTableData, flags, tbl, false)
```

Comment:
Why ignore the newly added result value?

Reply:
TiDB does not hold a lastInsertID for update statement. But MySQL does. I think we does not need a lastInsertID for update statement, because the behavior of MySQL is really buggy.